src/site/asciidoc/index.adoc

```diff
@@ -34,6 +36,20 @@ The project is actively maintained by a link:team.html[team] of several voluntee
 
 We share below some highlights from Log4j features.
 
+[source,java]
+----
+import java.util.logging.LogManager;
+import java.util.logging.Logger;
+
+class Application {
+    private static final Logger logger = LogManager.getLogger(Application.class);
+
+    public void action() {
+        logger.info("Hello, World!");
```

Comment:
This is a controversial subject. For one, `getLogger()` does not need an argument – this is always handy. I think a majority of existing Java styles use upper-case for `static final`s, hence my correction. @ppkarwasz (or somebody else) might not agree with me on this. Anyway, my 2 cents.

```suggestion
    private static final Logger LOGGER = LogManager.getLogger();

    public void action() {
        LOGGER.info("Hello, World!");
```

Reply:
Personally, I always make exceptions for logger. All this uppercase LOGGER things make the code unreadable and ugly for my taste. I am also not sure why it would be better to remove the name of the logger, I am curious about this. I have learned otherwise.

Reply:
Passing the class is not necessary anymore, since, unlike SLF4J, Log4j API can derive the caller from the context. :wink: This is not just easy, less typing, but also avoids copy-paste mistakes.

Regarding `LOGGER`-vs-`logger`, as I indicated, it is a controversial.

Please resolve the conversation as you wish. But not needing a class in `getLogger()` is a _feature_ we should be advertising, not hiding, IMHO.

Reply:
I didn't know about this feature, but want to learn more. I have made this wrong in my book too.

I will adjust to the simplest possible code and also use the Java standards, even when it is not my personal taste.

Reply:
Fixed with 270a42a127604874e832f05863379dbc1deb25f0